pyfr/solvers/base/system.py

```diff
@@ -130,11 +130,14 @@ def _load_eles(self, rallocs, mesh, initsoln, nregs, nonce):
     def _load_int_inters(self, rallocs, mesh, elemap):
         key = f'con_p{rallocs.prank}'
 
-        lhs, rhs = mesh[key].astype('U4,i4,i1,i2').tolist()
-        int_inters = self.intinterscls(self.backend, lhs, rhs, elemap,
-                                       self.cfg)
-
-        return [int_inters]
+        try:
+            lhs, rhs = mesh[key].astype('U4,i4,i1,i2').tolist()
+            int_inters = [self.intinterscls(self.backend, lhs, rhs, elemap,
+                                            self.cfg)]
+        except KeyError:
+            int_inters = []
```

Comment:
Not sure this should be needed for any sane mesh.

Reply:
I think the reason why I did this was some weirdness that was happening with metis.

Reply:
This can only happen if a rank has no internal connectivity which is not a viable scenario (performance here would be non-existent for example).